Eatery/Controllers/EateryTabBarController.swift

```diff
@@ -15,10 +15,13 @@ class EateryTabBarController: UITabBarController {
     let eateriesSharedViewController = EateriesSharedViewController()
     let lookAheadViewController = LookAheadViewController()
     let brbViewController = BRBViewController()
+    let accountManager = BRBAccountManager()
```

Comment:
Remove this from `EateryTabBarController`; I know I mentioned that you should do this before but I had a different implementation in mind (although yours works as well). 

I think we can make the call to `queryCachedBRBData` (or whatever you decide to rename it to, if you take my suggestion) inside the `BRBViewController`. The idea being that there should only be one `BRBAccountManager` in existence at a time.